components/service/glean/src/test/java/mozilla/components/service/glean/storages/EventsStorageEngineTest.kt

```diff
@@ -350,4 +356,149 @@ class EventsStorageEngineTest {
             ) == snapshot.first().extra)
         assertEquals(1, testGetNumRecordedErrors(testEvent, ErrorType.InvalidValue))
     }
+
+    @Test
+    fun `continue recording events from disk`() {
```

Comment:
Can we add a test to make sure that store files are correctly deleted when getting a snapshot with `clear = true`?

Reply:
Good idea.  I'll just add this as part of the existing `clearStores` test.